luigi.cfg

```diff
@@ -2,7 +2,7 @@
 logging_conf_file = logging.cfg
 
 [resources]
-toil = 5
+toil = 1
```

Comment:
Is this required for the AWS setup? This will ruin part of the power of this pipeline, which is multiple simultaneous toil pipelines running at once. We should at least document this for users.

Reply:
Ooops I didn't mean to push that change.

Reply:
Sadly it is necessary if you're using autoscaling, though we could keep the default the way it is, and instruct the user to change it in the very rare cases they're using autoscaling. That's probably the best solution, because really it will be pretty much just us using autoscaling.

The reason this is necessary when autoscaling is that the autoscaler assumes it has exclusive control of the cluster, so multiple runs will destructively interfere with each other.

@diekhans and I went back and forth about this -- there is probably an "easy" fix on the toil side, but "easy" is still realistically the better part of a day of work, so god knows when that will happen.

Reply:
I am fine with this as long as we document it for non-cloud users.